docs/Makefile

```diff
@@ -18,6 +21,12 @@ submodule: ## initialize the docsy theme submodule
 	git submodule update --init --recursive
 	cd themes/docsy; git am ../../patches/0001-Make-docsy-js-files-static.patch; cd ../..
 
+htmltest: install-htmltest
+	bin/htmltest -c $(DOCS_FOLDER)/.htmltest.yml -s $(DOCS_FOLDER)/public
+
+install-htmltest:
+	curl https://htmltest.wjdp.uk | bash
```

Comment:
A different installation method, particularly one that was pinned to a release version would be nice

Reply:
This always installs the latest release version, let me see if there's an argument to set the release version.

Reply:
Done